docker/release/config/opensearch/opensearch-docker-entrypoint-2.x.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
```

Comment:
These version scripts are very similar. Can one .sh call/include another? Would be nice if it wasn't a big copy paste with some small changes.